### DISCUSSION TITLE
[d2cpp]: d2cpp内容设计及大纲目录讨论

### DISCUSSION THREAD

**Discussion by Sunrisepeak** [d2learn/.github#1] [Ideas]
现在d2cpp项目的框架已经初步搭建完成, 所以对于d2cpp的内容需要进一步讨论, 形成一个大纲。最后在进行内容填充

- 有哪些需要特别注意的点
- 哪些内容是需要添加的以及添加到哪一个部分
- 内容思路
- 初学者视角中有哪些痛点
- 代码练习设计的点子

**大家可以在该讨论贴中进行交流讨论, 最后会进行汇总**

![image](https://github.com/user-attachments/assets/9656c151-a39d-4864-bc59-9fd95ef15276)

![image](https://github.com/user-attachments/assets/0e168a84-ccfc-4f1f-a2c1-4af0cc847004)

**项目:** [https://github.com/d2learn/d2cpp](https://github.com/d2learn/d2cpp)
---

**一些好的想法&点子近期汇总**

- xxx

**Comment by 1708737115** [comment]
就是大家觉得如果讲cpp多线程来作为一个章节的话，在章节前面我们介绍多线程的各个知识点，章节最后 实现一个线程池怎么样

**Comment by Sunrisepeak** [comment]
## 标准库 和 语言的语法和特性 是分开讲比较呢, 还是柔和到一起讲比较合适?
